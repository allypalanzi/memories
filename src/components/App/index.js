import React from 'react'
import styles from './index.scss'
import Game from '../Game/Game'

const App = () => (
  <main className={styles.app} role='main'>
    <h1 className={styles.header}>✨ Memory ✨</h1>
    <p className={styles.body}>Select a card and find it's pair!</p>
    <Game />
  </main>
)

export default App
