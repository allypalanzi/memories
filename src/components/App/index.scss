.header {
  font-size: 2em;
  margin-bottom: .5em;
}
.app {
  padding: 2em;
  font-family: sans-serif;
  text-align: center;
  max-width: 800px;
  margin: 0 auto;
}
