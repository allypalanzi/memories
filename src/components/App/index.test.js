import React from 'react'
import ReactDOM from 'react-dom'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import App from './'

jest.mock('../../client/services')

describe('<App/ >', () => {
  const renderedComponent = shallow(<App />)

  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<App />, div)
  })

  it('renders correctly', () => {
    expect(toJson(renderedComponent)).toMatchSnapshot()
  })
})
