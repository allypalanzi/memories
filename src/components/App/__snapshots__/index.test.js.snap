// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/ > renders correctly 1`] = `
<main
  className="l-wrapper"
  role="main"
>
  <h1>
    <span
      aria-label="sparkle emoji"
      role="img"
    >
      ✨
    </span>
    Memory
    <span
      aria-label="sparkle emoji"
      role="img"
    >
      ✨
    </span>
  </h1>
  <p>
    Select a card and find the pair!
  </p>
  <Game />
</main>
`;
